paying_for_college/templates/worksheet.html

```diff
@@ -2680,93 +2679,111 @@ <h2 class="step_heading">
-                            <h3 class="option_heading">
-                                Make sure you don’t have to take classes twice.
-                            </h3>
-                            <p>
-                                <a href="http://www.consumer.ftc.gov/articles/0395-choosing-college-questions-ask" target="_blank">Ask if credits from your school transfer
-                                </a> to degrees earned at other schools.
-                                <span id="option_completion-rate-content">
-                                <span id="option_completion-rate" data-financial="gradRate"
-                                data-percentage_value="true">[XX]</span>% of
-                                students who start at this school finish here.
-                                </span>
-                                In the case you decide to finish your
-                                education somewhere else, it’s important to
-                                know if the hard work you’ve put into your
-                                degree will be honored at another school.
-                            </p>
-                        </section>
-                        <section class="option option__explore-schools">
-                            <h3 class="option_heading">
-                                Explore other schools that have similar
-                                programs.
-                            </h3>
-                            <p>
-                                Review your total cost of attendance and
-                                determine if a less expensive school option
-                                (such as a community college) might also meet
-                                your educational needs.
-                                <a href="https://collegescorecard.ed.gov/"
-                                class="scorecard-link"
-                                target="_blank">
-                                    See all schools
-                                </a>
-                                that offer the same type of program in your
-                                area.
-                            </p>
-                        </section>
-                        <section class="option option__take-action column-well
-                        column-well__emphasis">
+                        <div class="get-options__dynamic">
+                            <section class="option option__maximize-grants">
+                                <h3 class="option_heading">
+                                    Maximize all available grants and scholarships.
+                                </h3>
+                                <p>
+                                    Consider <a href="https://studentaid.ed.gov/sa/types/grants-scholarships"
+                                    target="_blank"> applying for additional
```

Comment:
Design Manual says that external links should not be set to "_blank". Maybe there's a reason why this is an exception?

https://cfpb.github.io/design-manual/page-components/links.html